.github/workflows/checks.yml

```diff
@@ -153,11 +153,13 @@ jobs:
       fail-fast: false
       matrix:
         node:
-          - 20.11.0
-          - 22.0.0
+          - 20.17.0
+          - 22.9.0
```

Comment:
For the record, these were bumped because `npm@11.0.0` is specified to be compatible with these two as the lowest supported version.